Bottom: 7977b851cea7928096be9a6b666416e1b1d1fa9b
Top:    c8f668a4a84f5b86d96e2efd6b649b19f4696d8c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 22:25:58 +0000

Refresh of play-with-locked-object

---

diff --git a/src/draw.c b/src/draw.c
index a606884..3cbb710 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -480,10 +480,10 @@ DrawEMark (ElementType *e, Coord X, Coord Y, bool invisible)
   hid_draw_set_color (Output.fgGC, invisible ? PCB->InvisibleMarkColor : PCB->ElementColor);
   hid_draw_set_line_cap (Output.fgGC, Trace_Cap);
   hid_draw_set_line_width (Output.fgGC, 0);
-  hid_draw_draw_line (Output.fgGC, X - mark_size, Y, X, Y - mark_size);
-  hid_draw_draw_line (Output.fgGC, X + mark_size, Y, X, Y - mark_size);
-  hid_draw_draw_line (Output.fgGC, X - mark_size, Y, X, Y + mark_size);
-  hid_draw_draw_line (Output.fgGC, X + mark_size, Y, X, Y + mark_size);
+  hid_draw_line (Output.fgGC, X - mark_size, Y, X, Y - mark_size);
+  hid_draw_line (Output.fgGC, X + mark_size, Y, X, Y - mark_size);
+  hid_draw_line (Output.fgGC, X - mark_size, Y, X, Y + mark_size);
+  hid_draw_line (Output.fgGC, X + mark_size, Y, X, Y + mark_size);
 
   /*
    * If an element is locked, place a "L" on top of the "diamond".
